dynamicEDT3D/include/dynamicEDT3D/dynamicEDTOctomap.h

```diff
@@ -40,36 +40,38 @@
 
 #include "dynamicEDT3D.h"
 #include <octomap/OcTree.h>
+#include <octomap/OcTreeStamped.h>
 
-/// A DynamicEDTOctomap object connects a DynamicEDT3D object to an octomap.
-class DynamicEDTOctomap: private DynamicEDT3D {
+/// A DynamicEDTOctomapBase object connects a DynamicEDT3D object to an octomap.
+template <class TREE, class NODE>
```

Comment:
Is the `NODE` template parameter necessary? Where needed, you could use `typename TREE::NodeType` instead. Would make the interface a bit more lightweight.